spec/person_spec.rb

```diff
@@ -0,0 +1,7 @@
+require './person'
+
+describe Person.new 23, 'Abdu' do
+  it { is_expected.to have_attributes(age: 23) }
+  it { is_expected.to have_attributes(name: 'Abdu') }
+  it { is_expected.to have_attributes(parent_permission: true) }
+end
```

Comment:
- [ ] Good work writing your unit tests, but you forgot to test methods in the class, methods like `can_use_services`, `correct_name`, `add_rental` also test `default value for the optional name parameter if it is not provided`